front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx

```diff
@@ -1,22 +1,22 @@
 import { useEffect, useMemo } from 'react';
 
-import { Select } from '@osrd-project/ui-core';
+import { Select, ComboBox } from '@osrd-project/ui-core';
 import { useTranslation } from 'react-i18next';
 import nextId from 'react-id-generator';
 
 import type { SearchResultItemOperationalPoint } from 'common/api/osrdEditoastApi';
 import useSearchOperationalPoint from 'common/Map/Search/useSearchOperationalPoint';
 import type { PathStep } from 'reducers/osrdconf/types';
 
-import StdcmSuggestions from './StdcmSuggestions';
-
 type StdcmOperationalPointProps = {
   updatePoint: (pathStep: PathStep | null) => void;
   point: PathStep | null;
   opPointId: string;
   disabled?: boolean;
 };
 
+export type Option = { label: string; value: string; uic: number };
```

Comment:
Do we use it outside of this file ?

Reply:
It was used before in a helper not anymore, nice remark, resolved :)